data/hai/hai reveal 5 philosophers.txt

```diff
@@ -853,7 +836,7 @@ mission "Hai Reveal [B12] Devil-Hide Battle"
 			`To complete this mission, you must land with Choot'k, Teeneep, Danforth and Wallace's ships all present in the system and intact...`
 	on enter "Devil-Hide"
 		conversation
-			`Teeneep sends out a message to the fleet informing you that she is engaging the electronic warfare baffling and reminds you that everyone needs to get onto the ground intact if this is to go well. Choot'k's Shield Beetle has red engine flares; the others have blue. So if he gets into trouble you should be able to tell at a distance. The baffling won't work forever, so get to it.`
+			`Teeneep sends out a message to the fleet informing you that she is engaging the electronic warfare baffling and reminds you that everyone needs to get onto the ground intact if this is to go well. In case Choot'k gets into trouble, here's how to recognize his Shield Beetle from a  distance: it is reinforced with a darkened hull and has red engine flares while the others have blue. The baffling won't work forever, so get to it.`
```

Comment:
General Choot'k's design should be strong enough so he can handle the incoming pirates on his own, without help from escorts, for a few minutes. (Unless you nuke the area.) That's how I tuned the ship design.